packages/help-center/src/components/help-center-odie.tsx

```diff
@@ -122,6 +122,10 @@ export function HelpCenterOdie( {
 		[]
 	);
 
+	const isTestUser = useMemo( () => {
+		return currentUser?.username?.startsWith( 'e2eflowtesting' ) ?? false;
```

Comment:
Not sure if this works for our case, but worth a try.
https://github.com/Automattic/wp-calypso/blob/e1f240e1fb5a4083592a787b3e6a4f4c9f553692/apps/wpcom-block-editor/src/utils.js#L89

Reply:
Yeah, I was hoping there' a more "canonical" way of detecting we're running automated tests. I wonder if just detecting if we're in the `test` env is enough? https://github.com/Automattic/wp-calypso/blob/trunk/config/test.json But I'm not sure if that env is used for all tests - since I'd also assume that for E2E tests we would want to be as close to production as possible, so maybe `test` env is only for unit tests or something?

Reply:
https://github.com/Automattic/wp-calypso/blob/08ae7eb0ad31689ac63b5098d896a21d583f013c/packages/calypso-e2e/src/jest-playwright-config/playwright-config.ts#L45